### ISSUE TITLE
low mAP, high classification loss in eval - object detection resnet 50

### ISSUE THREAD

**Issue by qazalkz** [tensorflow/models#11207]
Hi guys. I have about 4000 images and Im using resnrt50 640*640 with 70000 steps. but after all Im getting low mAP. could you please give me some advice.

p.n: also I divide original learning rate to my batch size

pipeline_config.model.ssd.**num_classes** = 1
pipeline_config.train_config.**batch_size** = 64//8
pipeline_config.train_config.fine_tune_checkpoint = "checkpointmodels/retina101/ssd_/checkpoint/ckpt-0"
pipeline_config.train_config.fine_tune_checkpoint_type = "detection"
pipeline_config.train_input_reader.label_map_path= LABELMAP
pipeline_config.train_config.optimizer.momentum_optimizer.learning_rate.**cosine_decay_learning_rate.warmup_learning_rate=0.013333000242710114/8**
pipeline_config.train_config.optimizer.momentum_optimizer.learning_rate.**cosine_decay_learning_rate.learning_rate_base=0.03999999910593033/8**

![Screenshot from 2024-05-11 12-11-54](https://github.com/tensorflow/models/assets/69617228/87f5211e-e635-4174-8625-68ab10f01d73)
on 20000 stept eval 
![Screenshot from 2024-05-08 13-29-28](https://github.com/tensorflow/models/assets/69617228/b9a9630a-1c2c-462a-a372-98179e6ac860)
on 70000 steps 
![Screenshot from 2024-05-11 12-14-44](https://github.com/tensorflow/models/assets/69617228/72a3d1ea-7130-498e-8bf5-211d21cae4fe)
![Uploading Screenshot from 2024-05-11 12-17-28.png…]()
also you can see that classification loss is high on eval output

**Comment by Sam-Seaberry** [comment]
The model might be overfitting. Try increasing the learning rate and adding image augmentation.

**Comment by qazalkz** [comment]
thanks. I'll do it and share the results.

**Comment by laxmareddyp** [comment]
Hi @qazalkz,

Sorry for the late response and Please work out on  @Sam-Seaberry suggestions,it might solve the issue .This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-models) and [TensorFlow Forum](https://discuss.tensorflow.org/) since it is not a bug or feature request. There is also a larger community that reads questions there.

Thanks for understanding.
